core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityModel.scala

```diff
@@ -54,3 +54,13 @@ case class CompactEntityRefRecord(id: Long, name: String, entityType: String)
 case class RefPointerRecord(fromId: Long, toId: Long)
 
 case class KeysRecord(id: Long, workspaceId: UUID, entityType: String, attributeKeys: String, lastUpdated: Timestamp)
+
+case class EntityTypeAndAttributeKey(
+  entityType: String,
+  attributeKey: String
+)
```

Comment:
optional: use `AttributeName` instead of `String` for the attributeKey for strong typing to avoid any mixups between the two properties